bci_tester/data.py

```diff
@@ -305,9 +305,7 @@ def create_BCI(
     # only try to grab the mark from the build tag for containers that are
```

Comment:
so this comment is obsolete now? what made it obsolete?

Reply:
it is still correct, e.g. for 15.4 & 15.3. We don't have markers in the pyproject.toml for the sp3 & sp4 language containers (that don't exist anymore)

Reply:
ok, got it, so this if condition is only for nonbase containers.